io/net/multiport_stress.py.data/multiport_stress.yaml

```diff
@@ -0,0 +1,3 @@
+host_interfaces: "enP3p3s0f0,enP3p3s0f1"
+peer_ips: "10.68.68.26,90.90.90.102"
+count: 
```

Comment:
In the code, the default value for count is set to 1000. SO can the Readme for this script be updated to say that the count is 1000?